src/main/java/slimeknights/tconstruct/smeltery/data/SmelteryRecipeProvider.java

```diff
@@ -519,6 +528,10 @@ private void addSmelteryRecipes(Consumer<FinishedRecipe> consumer) {
                         .addByproduct(TinkerFluids.moltenGlass.result(FluidValues.GLASS_PANE))
                         .addByproduct(TinkerFluids.moltenIron.result(FluidValues.INGOT / 3))
                         .save(consumer, location(meltingFolder + "lantern"));
+    MeltingRecipeBuilder.melting(NoContainerIngredient.of(TinkerSmeltery.searedCastingTank), TinkerFluids.moltenCopper, FluidValues.INGOT, 2.5f)
+                        .addByproduct(TinkerFluids.searedStone.result(FluidValues.BRICK * 4))
+                        .addByproduct(TinkerFluids.moltenGlass.result(FluidValues.GLASS_BLOCK * 4))
```

Comment:
This should be 2 ingots copper, 3 blocks glass now, since the above recipe changed.